src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests.cs

```diff
@@ -752,12 +756,182 @@ class C
             Assert.True(semanticModel.TryGetSpeculativeSemanticModel(attrSyntax.ArgumentList.Position, newAttrSyntax, out var speculativeModel));
 
             var speculativeOperation = speculativeModel.GetOperation(newAttrSyntax);
-            // note: this operation tree should contain a constant string "M0" instead of null.
-            // this should ideally be fixed as part of https://github.com/dotnet/roslyn/issues/53618.
             VerifyOperationTree(comp, speculativeOperation, @"
-IOperation:  (OperationKind.None, Type: Attr) (Syntax: 'Attr()')
-  Children(1):
-      IDefaultValueOperation (OperationKind.DefaultValue, Type: System.String, Constant: null, IsImplicit) (Syntax: 'Attr()')
+IAttributeOperation (OperationKind.Attribute, Type: null) (Syntax: 'Attr()')
+  IObjectCreationOperation (Constructor: Attr..ctor([System.String s = null])) (OperationKind.ObjectCreation, Type: Attr, IsImplicit) (Syntax: 'Attr()')
+    Arguments(1):
+        IArgumentOperation (ArgumentKind.DefaultValue, Matching Parameter: s) (OperationKind.Argument, Type: null, IsImplicit) (Syntax: 'Attr()')
+          ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: ""M0"", IsImplicit) (Syntax: 'Attr()')
+          InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+          OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+    Initializer:
+      null
+");
+        }
+
+        [Fact]
+        public void TestAttributeCallerInfoSemanticModel_Method_Speculative2()
```

Comment:
I didn't understand what this was testing versus the original. Does it matter that `Method_Speculative` uses a string literal and `Method_Speculative2` uses a constant-valued interpolated string?

Reply:
I don't recall why did I add this test :(
I think it doesn't harm anyway.

Reply:
Works for me. Thanks